server/chat-plugins/poll.ts

```diff
@@ -454,9 +515,36 @@ export const commands: ChatCommands = {
 		`/poll results - Shows the results of the poll without voting. NOTE: you can't go back and vote after using this.`,
 		`/poll display - Displays the poll`,
 		`/poll end - Ends a poll and displays the results. Requires: % @ # & ~`,
+		`/deletequeue [number] - deletes poll with corresponding number from the queue.`,
+		`/clearqueue - deletes the queue of polls. Requires: % @ # & ~`,
+		`/viewqueue - view the queue of polls in the room. Requires: % @ # & ~`,
 	],
 };
 
+export const pages: PageTable = {
+	pollqueue(args, user) {
+		this.extractRoom();
+		const room = Rooms.get(args[0]) as ChatRoom | GameRoom;
+		let buf = `<div class = "pad"><strong>Queued polls</strong>`;
+		buf += `<button class="button" name="send" value="/join view-pollqueue-${room.roomid}" style="float: right">`;
+		buf += `<i class="fa fa-refresh"></i> Refresh</button><br />`;
+		if (!room.queuedActivity!.length) {
+			buf += `<hr/ ><strong>No polls queued.</strong></div>`;
+			return buf;
+		}
+		for (const poll of room.queuedActivity!) {
+			const num = room.queuedActivity?.indexOf(poll);
+			const button = (
+				`<button class="button" name="send" value="/poll deletequeue ${num},${room.roomid}">` +
+				`${num} in queue (press to delete)</button>`
```

Comment:
PS bans "click here" and similar, which this counts as.

Why not just `#${num} in queue <button>Delete</button>`?